README.md

```diff
@@ -119,9 +121,26 @@ alb_authenticate_oidc = {
 
 Read more in [this post](https://medium.com/@sandrinodm/securing-your-applications-with-aws-alb-built-in-authentication-and-auth0-310ad84c8595).
 
-If you are using GitHub, you may allow it to trigger webhooks without authentication on ALB:
 
+#### AWS Cognito
+
+The AWS Cognito service allows you to define SAML providers (e.g., GSuite). The Atlantis ALB can then be configured to require SAML authentication. To enable this, specify the following arguments containing attributes for your Cognito configuration.
+
+```hcl
+alb_authenticate_cognito = {
+  user_pool_arn               = "arn:aws:cognito-idp:us-west-2:1234567890:userpool/us-west-2_aBcDeFG"
+  cognito_user_pool_client_id = "clientid123"
+  cognito_user_pool_domain    = "sso.your-corp.com"
+}
 ```
+
+Read more in [this post](https://medium.com/@alsmola/alb-authentication-with-g-suite-saml-using-cognito-858e35564dc8) and a helpful [SAML Cognito Terraform module](https://github.com/alloy-commons/alloy-open-source/tree/master/terraform-modules/gsuite-saml-cognito).
```

Comment:
medium.com is a paid service and not everyone can read that post. It would be great if you include relevant information into README of your module so that everything is in one place. And remove the link from this module to the blog post (unless you can find it available on free resource).

Reply:
I tried cleaning up this section and pointed to the example usage for the gsuite-saml-cognito module. Let me know if you have other thoughts.

Reply:
Have you pushed your changes? I don't see them.

Reply:
sigh. Sorry about that. They should be pushed now.